### ISSUE TITLE
Publish docs instead of pdf user manual

### ISSUE THREAD

**Issue by jonhoo** [sslab-gatech/opensgx#29]
It would be really useful to have proper documentation for OpenSGX. The user manual/tutorial is a good start, but it'd be great if it were in a more digestible format such as [ReadTheDocs](https://readthedocs.org/), or simply as regular text-based files. I don't see a particularly good reason for serving it as a PDF?

**Comment by johnmwshih** [comment]
I also agree with using text-based instead of PDF. Probably because there are some graphics. I think we can simply make a text-based version.

**Comment by jonhoo** [comment]
That'd be great! The best would be if the documentation was in the code and then generated using something like [Sphinx](http://www.sphinx-doc.org/en/stable/), which is also supported by [ReadTheDocs](https://read-the-docs.readthedocs.org/en/latest/getting_started.html), so you can get nicely formatted, indexed, cross-linked, and searchable docs.

**Comment by jonhoo** [comment]
6cc1fe695b61bd34eed199402b8ea575a6fe49ed is a step in the right direction.
